modules/ROOT/pages/ogm/installation.adoc

```diff
@@ -187,11 +191,18 @@ export function comparePassword(plainText, hash) {
 
 [NOTE]
 ====
-This code for the util functions `createJWT` and `comparePassword` is an example. 
-You will likely need to adjust it to suit your use case.
+The code for the util functions `createJWT` and `comparePassword` is an example. 
+Adjust it to suit your use case.
 ====
 
-Back in the command line, run the following command to start your server:
+Install the additional dependencies:
+
+[source, bash, indent=0]
+----
+npm install bcrypt jsonwebtoken
+----
+
+Back on the command line, run the following command to start your server:
```

Comment:
This should be above the code block using the `bcrypt` dependency.

Reply:
just the "back on the command line", right?

Reply:
I mean the whole "Install the additional dependencies" part, because typically speaking you perform the necessary installation before you write the code.

Reply:
rearranged section a bit